server/routes/api/v1/petTypesRouter.js

```diff
@@ -12,6 +12,16 @@ petTypesRouter.get("/", async (req, res) => {
     console.error(errors)
     res.status(500).json({ errors })
   }
-}) 
+})
 
-export default petTypesRouter
\ No newline at end of file
+petTypesRouter.get("/:id", async (req, res) => {
+  try {
+    const petByType = await Pet.findById(req.params.id)
+    res.status(201).json({ petByType })
+  } catch (errors) {
+    console.error(errors)
+    res.status(500).json({ errors })
+  }
+})
+
```

Comment:
Great! Let's just change that `201` status to a `200` -- `201` means "created", but we're not creating anything here.